openpower/package/pkg-versions.mk

```diff
@@ -168,6 +170,11 @@ cd "$$(BR2_EXTERNAL_OP_BUILD_PATH)"; (git describe --always --dirty || echo "unk
 	>> $$($$(UPPER_CASE_PKG)_VERSION_FILE); \
 fi
 
+# Flag whether op-build is production signed
```

Comment:
Instead of having op-build have "knowledge" of methods of signing (or, indeed, keysets), I'd prefer a Linux style EXTRAVERSION to openpower/package/pkg-versions.mk, something to be appended to the top line in VERSION?

That way it becomes a bit more flexible, and if someone had several sets of keys they could use it for that, or adding anything else one might want.

This does kind of tie what you do in op-build to how you end up signing it (if you use EXTRAVERSION for that)... and maybe we should have a better (software) solution for that, I don't think there's a way to solve that elegantly simply by having things in VERSION.